src/cmd/linuxkit/push_vcenter.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// Process the push arguments and execute push
+func pushVCenter(args []string) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	var newVM vmConfig
+
+	flags := flag.NewFlagSet("vCenter", flag.ExitOnError)
+	invoked := filepath.Base(os.Args[0])
+
+	newVM.vCenterURL = flags.String("url", os.Getenv("VCURL"), "URL in the format of https://username:password@host/sdk")
+	newVM.dsName = flags.String("datastore", os.Getenv("VMDATASTORE"), "The Name of the DataStore to host the image")
+	newVM.networkName = flags.String("network", os.Getenv("VMNETWORK"), "The VMware vSwitch the image will use")
+	newVM.vSphereHost = flags.String("hostname", os.Getenv("VMHOST"), "The Server that will host the image")
+	newVM.path = flags.String("path", "", "Path to a specific image")
```

Comment:
Are these standard environment variables or made up?  They seem to be inconsistently named and I would prefer if they had a common prefix...